components/OCFBanner.tsx

```diff
@@ -72,3 +59,178 @@ export const OCFCollectivePageBanner = ({ collective, LoggedInUser }) => {
     description: <Message isCentered collective={collective} params={params.toString()} />,
   };
 };
+
+type OCFBannerProps = {
+  collective: Pick<Account, 'slug'>;
+  hideNextSteps?: boolean;
+};
+
+export function OCFBanner(props: OCFBannerProps) {
+  const { LoggedInUser } = useLoggedInUser();
+  const query = useQuery<OcfTransitionBannerQuery, OcfTransitionBannerQueryVariables>(
+    gql`
+      query OCFTransitionBanner($slug: String!) {
+        account(slug: $slug) {
+          id
+          name
+          slug
+          ... on AccountWithHost {
+            host {
+              id
+              legacyId
+            }
+          }
+          oldAccount: members(role: [CONNECTED_ACCOUNT], limit: 1) {
+            totalCount
+            nodes {
+              id
+              role
+              publicMessage
+              description
+              account {
+                id
+                name
+                slug
+                imageUrl
+                ... on AccountWithHost {
+                  host {
+                    id
+                    legacyId
+                  }
+                }
+              }
+            }
+          }
+
+          newAccount: memberOf(role: [CONNECTED_ACCOUNT], limit: 1) {
```

Comment:
@hdiniz This is not a reliable way to look for the new account, `CONNECTED_ACCOUNT` is used elsewhere for different purposes

Reply:
Hey yes  you  right  just give little time  I do my best to make it work